<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayerPanel</class>
 <widget class="QWidget" name="LayerPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>129</width>
    <height>809</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="Frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>129</width>
     <height>809</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="Layer_del">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>4</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resources/Layers/layer_del.png</normaloff>Resources/Layers/layer_del.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Selection_del">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>4</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resources/Selections/selec_del.png</normaloff>Resources/Selections/selec_del.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Layer_fuse">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>4</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resources/Layers/layer_fuse.png</normaloff>Resources/Layers/layer_fuse.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Layer_new">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>4</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resources/Layers/layer_new.png</normaloff>Resources/Layers/layer_new.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="Layer_frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>32</y>
      <width>129</width>
      <height>777</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
